debian/rules

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/make -f
+
+DEB_DESTDIR := debian/node-red-contrib-revpi-nodes
+
+%:
+	dh $@ --with nodejs
+
+override_dh_auto_test:
+
+override_dh_auto_install:
+	dh_auto_install --buildsystem=nodejs
+	mkdir -p $(DEB_DESTDIR)/usr/lib/node_modules
+	mv $(DEB_DESTDIR)/usr/share/nodejs/node-red-contrib-revpi-nodes \
+	   $(DEB_DESTDIR)/usr/lib/node_modules
+	mkdir -p $(DEB_DESTDIR)/usr/lib/node_modules/node-red-contrib-revpi-nodes/node_modules
+	mv $(DEB_DESTDIR)/usr/lib/node_modules/node-red-contrib-revpi-nodes/ws \
+	   $(DEB_DESTDIR)/usr/lib/node_modules/node-red-contrib-revpi-nodes/node_modules
+	rmdir $(DEB_DESTDIR)/usr/share/nodejs
```

Comment:
`/usr/lib/node_modules` seems to be the correct path for npm's global `node_modules` directory on the RevPi. The path can be retrieved with `npm root -g`, and this shows a different prefix (`/usr/local/lib/node_modules`) for my laptop.

**Assuming** that Node-RED looks for additional nodes in npm's global `node_modules` directory this path can't be hard coded like it is here and needs to be adjusted. Additionally, Debian specifies directories where node modules should live here: https://wiki.debian.org/Javascript/Nodejs#Packaging_modules
While testing these directories Node-RED didn't pick up the nodes, though.